src/Uno.UI.RuntimeTests/MUX/Microsoft_UI_Xaml_Controls/ProgressRing/ProgressRingTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Threading.Tasks;
+using MUXControlsTestApp.Utilities;
+using Private.Infrastructure;
+using Uno.UI.RuntimeTests.Helpers;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.UI.RuntimeTests.MUX.Microsoft_UI_Xaml_Controls.ProgressRingTests;
+
+[TestClass]
+public class ProgressRingTests
+{
+	[TestMethod]
+	[DataRow(true)]
+	[DataRow(false)]
+	public async Task ProgressRingDefaultHeightShouldBe32(bool useFluent)
+	{
+		using (useFluent ? StyleHelper.UseFluentStyles() : null)
+		{
+			var grid = new Grid();
+			grid.Width = 100;
+			grid.Height = 100;
+			var progressRing = new Microsoft.UI.Xaml.Controls.ProgressRing() { IsActive = true };
+			grid.Children.Add(progressRing);
+			RunOnUIThread.Execute(() => TestServices.WindowHelper.WindowContent = grid);
+
+			await TestServices.WindowHelper.WaitForLoaded(progressRing);
+
+#if __MACOS__
+			Assert.AreEqual(16, progressRing.ActualHeight);
+			Assert.AreEqual(16, progressRing.ActualWidth);
```

Comment:
I don't understand why mac is different. But anyway that would be unrelated to the small change I made.

Reply:
Note: 16 is the min width/height